### PR TITLE
Issue #ED-1052: cleanup of ngx-infinite-scroll dependency

### PR DESCRIPTION
**Pull request by princegupta1131**
# SunbirdEd - Portal
---

### Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
---

#### Example
- [x] Applicable
- [ ] Not applicable

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/client/src/app/modules/discussion/discussion.module.ts [hunk 1]
```diff
@@ -28,9 +28,9 @@ export class DiscussionModule {
     if (!CsModule.instance.isInitialised) {
       this.csLibInitializerService.initializeCs();
     }
-    this.discussionEvents.telemetryEvent.subscribe((event) => {
-      this.discussionTelemetryService.logTelemetryEvent(event);
-    });
+    // this.discussionEvents.telemetryEvent.subscribe((event) => {
+    //   this.discussionTelemetryService.logTelemetryEvent(event);
+    // });
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/discussion/discussion.module.ts:33]: @princegupta1131 
remove the commented code if not required

File: src/app/client/src/app/modules/uci-admin/components/uci/uci.component.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<lib-lib-entry
+<!-- <lib-lib-entry
   [user]="userProfile" [baseUrl]="url" [blobUrl]="blobUrl" [botPhoneNumber]="uciBotPhoneNumber"
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/uci-admin/components/uci/uci.component.html:2]: @princegupta1131 
removed commented code if not required

File: src/app/client/src/app/modules/uci-admin/uci-admin.module.ts [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-import {UciModule} from '@samagra-x/uci-console';
+// import {UciModule} from '@samagra-x/uci-console';
 
 
 @NgModule({
   declarations: [UciComponent],
   imports: [
     UciAdminRoutingModule,
-    UciModule
+    // UciModule
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/uci-admin/uci-admin.module.ts:11]: @princegupta1131 
removed commented code if not required

File: src/app/client/src/app/modules/workspace/workspace.module.ts [hunk 4]
```diff
@@ -44,7 +44,7 @@ import { MatTooltipModule } from '@angular/material/tooltip';
     OrderModule,
     PlayerHelperModule,
     ContentSearchModule,
-    CollectionEditorLibraryModule,
+    // CollectionEditorLibraryModule,
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/workspace/workspace.module.ts:47]: @princegupta1131 
removed commented code if not required

**Review by rajeshkumaravel — CHANGES_REQUESTED** [review]

File: src/app/client/src/app/modules/discussion/discussion.module.ts (hunk 1 above)
Comment (princegupta1131) [reply src/app/client/src/app/modules/discussion/discussion.module.ts]: Removed the commeted line

File: src/app/client/src/app/modules/uci-admin/components/uci/uci.component.html (hunk 2 above)
Comment (princegupta1131) [reply src/app/client/src/app/modules/uci-admin/components/uci/uci.component.html]: Removed the commeted line

File: src/app/client/src/app/modules/uci-admin/uci-admin.module.ts (hunk 3 above)
Comment (princegupta1131) [reply src/app/client/src/app/modules/uci-admin/uci-admin.module.ts]: Removed the commeted line

File: src/app/client/src/app/modules/workspace/workspace.module.ts (hunk 4 above)
Comment (princegupta1131) [reply src/app/client/src/app/modules/workspace/workspace.module.ts]: Removed the commeted line

**Review by rajeevsathish — APPROVED** [review]

File: src/app/client/angular.json [hunk 5]
```diff
@@ -29,11 +29,7 @@
                 "input": "./node_modules/@project-sunbird/common-consumption/assets",
                 "output": "./assets/common-consumption"
               },
-              {
-                "glob": "**/*.*",
-                "input": "./node_modules/@project-sunbird/discussions-ui-v8/assets",
-                "output": "./assets/discussion-ui"
-              },
+
```
[the remark this answers is not in the document]
Comment (princegupta1131) [reply src/app/client/angular.json:32]: its json file : error Comments are not permitted in JSON.

**Review by rajeevsathish — APPROVED** [review]

**Pull request merged**